src/communication/network_interface_cl.c

```diff
@@ -10888,3 +10888,45 @@ flashback_get_loginfo (int trid, char *user, OID * classlist, int num_class, LOG
 #endif // CS_MODE
   return ER_NOT_IN_STANDALONE;
 }
+
+int
+plcsql_transfer_file (const std::string & input_file, const bool & verbose, std::string & output_file,
+		      std::string & sql)
+{
+#if defined(CS_MODE)
+  int rc = ER_FAILED;
+  packing_packer packer;
+  cubmem::extensible_block eb;
+  char *ptr = NULL;
+
+  char *data_reply = NULL;
+  int data_reply_size = 0;
+
+  packer.set_buffer_and_pack_all (eb, verbose, input_file);
+
+  OR_ALIGNED_BUF (2 * OR_INT_SIZE) a_reply;
+  char *reply = OR_ALIGNED_BUF_START (a_reply);
+  int req_error = net_client_request2 (NET_SERVER_PLCSQL_TRANSFER_FILE, eb.get_ptr (), (int) packer.get_current_size (),
+				       reply, OR_ALIGNED_BUF_SIZE (a_reply), NULL, 0, &data_reply, &data_reply_size);
+  if (!req_error)
+    {
+      ptr = or_unpack_int (reply, &data_reply_size);
+      ptr = or_unpack_int (ptr, &rc);
```

Comment:
Is there no need to check the rc obtained here?

Reply:
The `rc` is dummy data here. I don't know why but It always returns NO_ERROR.